hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -53,6 +53,24 @@
     <tag>OZONE, CONTAINER, MANAGEMENT</tag>
     <description>The ipc port number of container.</description>
   </property>
+  <property>
+    <name>dfs.container.ratis.datastream.ipc</name>
+    <value>9890</value>
+    <tag>OZONE, CONTAINER, RATIS, DATASTREAM</tag>
+    <description>The datastream ipc port number of container.</description>
+  </property>
+  <property>
+    <name>dfs.container.ratis.datastream.request.threads</name>
+    <value>20</value>
+    <tag>OZONE, CONTAINER, RATIS, DATASTREAM</tag>
```

Comment:
Cab we move these settings to DatanodeRatisServerConfig and clinet config keys separately?

Reply:
Had moved datastream.request.threads and datastream.write.threads to DatanodeRatisServerConfig.  The rest of the configurations are similar to [those related to Container](https://github.com/apache/ozone/pull/2452/files#diff-148cad3e4555d212af7e3c58a71fb17b2782587d89659f666227c6d92d7d898cR77), and I keep them in this place.